spec/index.html

```diff
@@ -80,6 +80,13 @@
       .separated tbody tr td { border:1px solid black; text-align: center; }
       .separated tbody tr td:first-child, .separated tbody tr th:first-child {max-width: 220px; overflow-wrap: anywhere;}
     }
+
+    table.cp-definitions { border-collapse:collapse; background-color: #DDDDFF}
+
+    table.cp-definitions,
+    table.cp-definitions th,
+    table.cp-definitions td { border:1px solid black; padding:0.2em; }
```

Comment:
I suggest similar `padding`, especially `padding-left` and `padding-right`, be added to all table CSS. Generally speaking, I also suggest `text-align: center` for all table headers.
```suggestion
    table.cp-definitions th { text-align: center; padding:0.2em; padding-left: 0.3em; padding-right: 0.3em; },
    table.cp-definitions td { border:1px solid black; padding:0.2em; padding-left: 0.3em; padding-right: 0.3em; }
```

Reply:
Addressing CSS table styling for all tables in all specs is a subject for https://github.com/w3c/rdf-star-wg/issues/61, but we can do that here (and should retro-fit to other such cp-definitions tables in other specs).